apps/src/feedback.js

```diff
@@ -860,15 +863,30 @@ FeedbackUtils.prototype.getGeneratedCodeString_ = function() {
 };
 
 /**
- *
+ * Generates a "show code" div with a description of what code is.
+ * @param {Object} [options] - optional
+ * @param {string} [options.generatedCodeDescription] - optional description
+ *        of code to put in place instead of the default
+ * @returns {Element}
+ * @private
  */
-FeedbackUtils.prototype.getGeneratedCodeElement_ = function() {
+FeedbackUtils.prototype.getGeneratedCodeElement_ = function(options) {
+  options = options || {};
+
   var codeInfoMsgParams = {
     berkeleyLink: "<a href='http://bjc.berkeley.edu/' target='_blank'>Berkeley</a>",
     harvardLink: "<a href='https://cs50.harvard.edu/' target='_blank'>Harvard</a>"
   };
 
-  var infoMessage = this.studioApp_.editCode ?  "" : msg.generatedCodeInfo(codeInfoMsgParams);
+  var infoMessage = "";
+  if (this.studioApp_.editCode) {
+    infoMessage = ""
+  } else if (options.generatedCodeDescription) {
+    infoMessage = options.generatedCodeDescription;
+  } else {
+    infoMessage = msg.generatedCodeInfo(codeInfoMsgParams);
+  }
+
```

Comment:
I wonder if we should instead have  `getGeneratedCodeDescription` that does all this infoMessage stuff, and then passes that string to `getGeneratedCodeElement_`. Thoughts?